### PR TITLE
Move CLI docs from getting_started to here

### PR DESCRIPTION
**Pull request by illusional**

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -4,28 +4,32 @@ CLI for interacting with the REMOTE analysis-runner (externally hosted `server/m
...
+1. Omitting the `--repository` parameter: use the repository of the local directory that you're in: (a) get the repository name from the git remote; (b) use the commit of HEAD (if the `--commit` parameter is omitted); (c) Make the script path relative to the root of the git repository.
+
+  ```bash
+  # cwd is the git directory root
+  cd path/to/script
+  analysis-runner \
+      --dataset <dataset> \
+      --description "Description of the run" \
+      --output-dir gs://<bucket> \
+      main.py and some arguments # becomes path/to/script/main.py and some arugments
```
Comment (lgruen) [inline cli/README.md:18]:
```suggestion
      main.py and some arguments # becomes path/to/script/main.py and some arguments
```

File: cli/README.md [hunk 2]
```diff
@@ -4,28 +4,32 @@ CLI for interacting with the REMOTE analysis-runner (externally hosted `server/m
...
+  ```
+
+1. Providing the `--repository REPO` parameter, making the script path relative to the git repository is disabled, and you must provide a commit hash too. For example:
```
Comment (lgruen) [inline cli/README.md:21]:
```suggestion
1. Providing the `--repository` parameter, making the script path relative to the git repository is disabled, and you must provide a commit hash too. For example:
```

File: cli/README.md [hunk 3]
```diff
@@ -4,28 +4,32 @@ CLI for interacting with the REMOTE analysis-runner (externally hosted `server/m
...
+
+  ```bash
+  # You must specify relative path from git root to script
+  analysis-runner \
+      --dataset <dataset> \
+      --description "Description of the run" \
+      --output-dir gs://<bucket> \
+      --repository hail-batch-test \
```
Comment (lgruen) [inline cli/README.md:29]:
```suggestion
      --repository <repository> \
```

**Review by lgruen — APPROVED** [review]
Thanks! Added some minor remarks.

**Comment by illusional** [comment]
Applied suggestions, and merging :)

**Pull request merged**